cli/cmd/dashboard.go

```diff
@@ -33,10 +37,26 @@ var dashboardCmd = &cobra.Command{
 			os.Exit(1)
 		}
 
+		client, err := newPublicAPIClient()
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "Failed to initialize API Client: %+v\n", err)
```

Comment:
i think we want to exit here? maybe same with the following error?

Reply:
also we may want to say "Conduit API client", just so we're clear which API we're talking about (i.e. not kubernetes).